.github/workflows/release.yml

```diff
@@ -49,26 +59,61 @@ jobs:
         if: ${{ fromJSON(matrix.config.os == 'windows-latest') }}
         env:
           OPENSSL_DIR: C:\Program Files\OpenSSL-Win64\
+          OPENSSL_NO_VENDOR: true
+
+      - name: "build for macos amd64"
+        run: make build
+        if: ${{ fromJSON(matrix.config.os == 'macos-latest') && 
+          fromJSON(matrix.config.arch == 'amd64') }}
+      
+      - name: "build for macos aarch64"
+        run: cargo build --manifest-path ./slight/Cargo.toml --release ${{ matrix.config.extraArgs }}
```

Comment:
could you please put the command in makefile?

Reply:
~didn't want to do it 'cause of the action specific argument